deliver/lib/deliver/options.rb

```diff
@@ -24,6 +24,11 @@ def self.available_options
                                      env_name: "DELIVER_APP_ID",
                                      description: "The app ID of the app you want to use/modify",
                                      is_string: false), # don't add any verification here, as it's used to store a spaceship ref
+        FastlaneCore::ConfigItem.new(key: :edit_live,
+                                     short_option: "-E",
```

Comment:
i wanted to use `-e` (edit) -> used, then `-l` (live edit) -> used - so i used `-E`, anyway i changed it, as the whole commandline argument thing is not perfect anyway in all tools 👍  changed to `-o`